lib/run_loop/otool.rb

```diff
@@ -0,0 +1,76 @@
+module RunLoop
+  # @!visibility private
+  #
+  # A class for interacting with otool
+  class Otool
+
+    # @!visibility private
+    attr_reader :path
+
+    # @!visibility private
+    def initialize(path)
+      @path = path
+
+      if !Otool.valid_path?(path)
+        raise ArgumentError,
+%Q{File:
+
+#{path}
+
+must exist and not be a directory.
+}
+      end
+    end
+
+    # @!visibility private
+    def to_s
+      "#<OTOOL: #{path}>"
```

Comment:
It could be @path, but since we are failing we can just use the argument.

This will work even if line the assignment on line 12 `@path = path` is moved.